Sources/PointFree/Blog/BlogPost.swift

```diff
@@ -29,6 +29,11 @@ This is the blurb to a mock blog post. This should just be short and to the poin
 text, no markdown.
 """,
   contentBlocks: [
+    .init(
+      content: "",
+      timestamp: nil,
+      type: .video(poster: "", sources: [])
```

Comment:
this is for tests, but wish I had a way to render some mock images in tests. we could do it via `src` data and svgs, but I'll save that for another time.